document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresTableIdentifier.java

```diff
@@ -0,0 +1,43 @@
+package org.hypertrace.core.documentstore.postgres;
+
+import java.util.Optional;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+
+@EqualsAndHashCode
+public class PostgresTableIdentifier {
```

Comment:
Isn't database a part of the table identifier? Or, it can be skipped because it's already a part of the connection configuration?

Reply:
Technically yes. Currently everything is in the scope of one database though, so we omit it. The schema is what we were using the implicit value for ("public").

Reply:
Makes sense. Worth mentioning that in the comment too.?